packages/scipy/patches/rename-_page_trend_test.patch

```diff
@@ -0,0 +1,34 @@
+From 6ffb169dbe331627d70da3f79f928057f1a4e163 Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Sun, 26 Dec 2021 07:34:40 -0800
+Subject: [PATCH] Rename _page_trend_test.py to prevent test unvendoring
+
+unvendor_tests will unvendor any file that ends in _test.py. 
```

Comment:
yes, that's unfortunate. Layer, we should probably come up with some way to add exceptions to that code..